skipruntime-ts/core/src/skipruntime_api.ts

```diff
@@ -25,6 +35,11 @@ export type Param =
  * `Loading`, or `Error`
  */
 
+export type Loadable<V extends TJSON, M extends TJSON> =
+  | Success<V, M>
+  | Loading<V, M>
+  | Error<V, M>;
```

Comment:
What are others' view on single letter type parameters versus longer role-indicating names in the public api?

Reply:
Agreed (maybe with the exception of K/V as key/value or T for the members of a collection)